xarray/util/generate_aggregations.py

```diff
@@ -313,21 +333,36 @@ def generate_method(self, method):
         for text in [
             self._dim_docstring.format(method=method.name, cls=self.cls),
             *(kwarg.docs for kwarg in method.extra_kwargs if kwarg.docs),
-            _KEEP_ATTRS_DOCSTRING,
+            _KEEP_ATTRS_DOCSTRING.format(
```

Comment:
Unfortunately I had to add this ugliness since we'd like `keep_attrs` to be strict `bool` (#3891).

To compensate, I cleaned up the special "see also" for `count` :)

Reply:
I haven't been in touch with the NameArray discussions so forgive a glib comment — but re #3891 — this would be a "once-in-a-library" opportunity to just keep attrs in aggregations, removing the option to each arg.

(Xarray could still handle them as it wished, so xarray's external interface wouldn't need to change immediately...)

Reply:
I agree. Current version deletes keep_attrs :)